driver/driver.go

```diff
@@ -30,7 +35,7 @@ func (b bigQueryDriver) Open(uri string) (driver.Conn, error) {
 
 	ctx := context.Background()
 
-	client, err := bigquery.NewClient(ctx, config.projectID)
+	client, err := bigquery.NewClient(ctx, config.projectID, option.WithScopes(config.scopes...))
```

Comment:
It is ok to always pass this option, or should it be passed only when the `scopes` query argument is present?

Reply:
If the passed scopes array is empty, then it doesn't take any effect. However, if it is clearer to pass it only when scopes param is present, I can change it. What do you think?

Reply:
If it doesn't have any effect then it should be all right to leave it like that

Reply:
I think it doesn't have any effect since I checked the source code and I see that the length of the slice is being checked as > 0 and otherwise it takes some default scopes, you can check it out here and let me know: https://github.com/googleapis/google-api-go-client/blob/a4ae94d5614a5efa03a18acb1508e38efe06d1b1/internal/settings.go#L60

Reply:
Great job checking that!